src/hooks/useVaultFactory.ts

```diff
@@ -41,7 +43,12 @@ export function useVaultFactory(orgId?: number) {
 
       const { receipt } = await sendAndTrackTx(
         () => contracts.vaultFactory.createApeVault(...args),
-        { showInfo, showError }
+        {
+          showInfo,
+          showError,
+          description: `Create ${type || customSymbol} Vault`,
+          chainId: contracts.chainId,
```

Comment:
@teeolendo @topocount once this is merged, you can add these two props to `sendAndTrackTx` for claim, withdraw, etc. to add them to the recent transactions list as well. (They'll already be added, but as "Unlabeled Transaction" with blank chain ID.)